torch/csrc/jit/python/init.cpp

```diff
@@ -158,6 +161,9 @@ class PythonSymIntNodeImpl : public c10::SymIntNodeImpl {
     return getPyObj().attr("__int__")().cast<int64_t>();
   }
 
+  // TODO: virtualize
+  SymFloat sym_float();
```

Comment:
On my local build in the branch I had to add `c10::` here. But looks ok here.

Reply:
we have

```
using c10::SymFloat;
```

in the file.